xbmc/input/KeyboardStat.cpp

```diff
@@ -32,6 +32,8 @@
 #include "threads/SystemClock.h"
 #include "utils/log.h"
 
+#define HOLD_THRESHOLD 250
```

Comment:
sorry, github gobbled my &lt;&gt;

the idea was to define a duration next to the modifier

&lt;start mod="longpress" duration="1000">ActivateWindow(Home)&lt;/guide>